### ISSUE TITLE
closeable v-chip doesn't seem to be working

### ISSUE THREAD

**Issue by oakdays** [johnleider/vue-materials#58]
While browsing the Chip section of the documentation, i noticed that the v-chip component with the close property is not being closed and seems to be visually not aligned as well. This seems to be an issue with a v-icon inside the div with the chip class. I think the component would work if the v-icon had the "close" class, guessing from the Materialize documentation.

```
<!-- Materialize documentation for a closeable chip -->
<div class="chip">
    Tag
    <i class="close material-icons">close</i>
</div>
```
However, there is code in the chip component regarding controlling the visibility of the close icon with an event and a variable. Shoudn't the component just show a v-icon with the "close" class without having to handle events and variables?
```
<!-- Chip component template -->
<div class="chip"
     v-if="!remove">
    <slot></slot>
    <v-icon v-if="close">close</v-icon>
</div>
```
```
<!-- Chip component template suggestion -->
<div class="chip">
    <slot></slot>
    <v-icon v-if="close" class="close">close</v-icon>
</div>
```

**Comment by alexanderstoev** [comment]
I've seen this too.
Minor change is needed in the v-icon component to include "close" prop and the v-chip component to send it.
The code you've proposed for the chip component will always set the close class.
This can be good or bad depending on the point of view :)

**Comment by oakdays** [comment]
Yeah it will always set but if you want the chip to be closeable you want the icon with the class.
I'm pretty sure @johnleider would appreciate any help. Maybe show a jsfiddle with the working code?

**Comment by johnleider** [comment]
Could you please provide a display of the code with jsfiddle, jsbin, codepen or something of the sort please.
